database/data.js

```diff
@@ -0,0 +1,44 @@
+
+
+
+//helper function
+//pick 5 random numbers between 1000-1100
+const getComparableItemsId = (productId) => {
+  //pick 5 random numbers between 1000-1100
+  //can't pick the same number as the product
+  let productIds = [];
+
+  for (let j = 0; j < 5; j++) {
+    let randomNum = Math.floor(Math.random() * (1100 - 1000));
+    randomNum = randomNum + 1000;
+    productIds.push(randomNum);
+  }
+  //check to see if all values are different than the comparing product
+  for (let k = 0; k < productIds.length; k++) {
```

Comment:
There may be an opportunity to prevent having a second for loop if randomNum === productId is checked within the first for loop (randomNum can be re-generated). It could shorten code by removing 1 for loop, and prevent having to run the function again to create a new array. This would probably only occur 5% of the time, so refactoring may in the end not make much difference.

Reply:
It may be worth adding edge cases for duplicate productIds within the same array. For example, when productId is 1000, comparableItems array could potentially be [1001, 1001...]. Again, the rate at which this would occur is low.